agent/agent.go

```diff
@@ -101,6 +101,7 @@ func (r *Agent) Run(recipe recipe.Recipe) (run Run) {
 		ctx         = context.Background()
 		getDuration = r.startDuration()
 		stream      = newStream()
+		dataCount   = 0
```

Comment:
@GrayFlash Let's call it `recordCount` ?

Reply:
Sure makes sense, will add it to the report table in `run.go` as the column named `Records` right??